From 3a854ce4e2309ffb5a9c6aa2a9cd8a5fa941a5f6 Mon Sep 17 00:00:00 2001
From: Daniel Ammann <daniel.ammann@bytesatwork.ch>
Date: Thu, 4 Apr 2024 11:12:13 +0200
Subject: [PATCH] configs: imx8mp_bytedevkit: Add cpuinfo and meminfo commands

These commands are needed for testing.

Change prompt for testing.

Signed-off-by: Daniel Ammann <daniel.ammann@bytesatwork.ch>
---
 configs/imx8mp_bytedevkit_defconfig | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/configs/imx8mp_bytedevkit_defconfig b/configs/imx8mp_bytedevkit_defconfig
index c8836445d1..c3aaa94bad 100644
--- a/configs/imx8mp_bytedevkit_defconfig
+++ b/configs/imx8mp_bytedevkit_defconfig
@@ -45,7 +45,8 @@ CONFIG_SYS_MMCSD_RAW_MODE_U_BOOT_SECTOR=0x300
 CONFIG_SPL_I2C=y
 CONFIG_SPL_POWER=y
 CONFIG_SPL_WATCHDOG=y
-CONFIG_SYS_PROMPT="u-boot=> "
+CONFIG_SYS_PROMPT="=> "
+CONFIG_CMD_CPUINFO=y
 # CONFIG_BOOTM_NETBSD is not set
 # CONFIG_CMD_EXPORTENV is not set
 # CONFIG_CMD_IMPORTENV is not set
@@ -53,6 +54,7 @@ CONFIG_CMD_ERASEENV=y
 CONFIG_CMD_NVEDIT_EFI=y
 CONFIG_CMD_CRC32=y
 CONFIG_CRC32_VERIFY=y
+CONFIG_CMD_MEMINFO=y
 CONFIG_CMD_MEMTEST=y
 CONFIG_CMD_CLK=y
 CONFIG_CMD_DFU=y
-- 
2.39.2

